telegram/_inline/inlinequeryresultgif.py

```diff
@@ -71,7 +71,8 @@ class InlineQueryResultGif(InlineQueryResult):
         gif_duration (:obj:`int`): Optional. Duration of the GIF in seconds.
         thumb_url (:obj:`str`): URL of the static (JPEG or GIF) or animated (MPEG4) thumbnail for
             the result.
-        thumb_mime_type (:obj:`str`): Optional. MIME type of the thumbnail.
+        thumb_mime_type (:obj:`str`): Optional. MIME type of the thumbnail, must be one of
+            ``'image/jpeg'``, ``'image/gif'``, or ``'video/mp4'``. Defaults to ``'image/jpeg'``.
```

Comment:
should we add this to constants? If so, just as plain strings or are there any object references?

Reply:
constants mime types could be an option although I'm wondering if there is a list of constants for mime types available somewhere outh there already … I don't quite get the second question.
Can be done in a separate PR though

Reply:
In the second question I meant that these lines just say ``'image/jpeg'``, but in constants.py I find a special reference to a mime type:
```
Bots can send :mimetype:`audio/ogg` files of up to 1MB in size
```

Reply:
See [this comment](https://github.com/python-telegram-bot/python-telegram-bot/pull/3404#discussion_r1056438950)